agent/pkg/proxy/config/config.go

```diff
@@ -1,5 +1,11 @@
 package config
 
+import (
+	meshConstants "github.com/kubeedge/edgemesh/common/constants"
```

Comment:
golang package import rule:
1. built-in pkg
2. third-party pkg
3. project pkg